spec/services/project_life_cycle_steps/update_service_spec.rb

```diff
@@ -165,6 +165,17 @@
       end
     end
 
+    context "for following phase with a start date only" do
+      let!(:phase) { create(:project_phase, project:, start_date: date, finish_date: date) }
+      let!(:following) { create(:project_phase, project:, start_date: date + 1, finish_date: nil, duration: 3) }
```

Comment:
duration should be `nil` here, probably easier implicitly:
```suggestion
      let!(:following) { create(:project_phase, project:, start_date: date + 1, finish_date: nil) }
```

Reply:
Having an unrelated duration makes sure we do not accidentally clear or re-calculate the duration while rescheduling with a start date only.